integreat_cms/static/src/js/media-management/component/edit-sidebar.tsx

```diff
@@ -247,7 +251,10 @@ const EditSidebar = ({
                                         className={cn(
                                             "w-full text-white text-center font-bold py-3 px-4 m-0 rounded",
                                             { "cursor-not-allowed bg-gray-500": isLoading },
-                                            { "bg-blue-500 hover:bg-blue-600": !isLoading }
+                                            { "bg-blue-500 hover:bg-blue-600": !isLoading },
+                                            {
+                                                hidden: !canReplaceFile,
+                                            }
```

Comment:
In this case, I'd also use `{!selectionMode && canReplaceFile && (` in line 247 instead of the hidden class.